drivers/disk/sdmmc_stm32.c

```diff
@@ -755,21 +796,20 @@ static int stm32_sdmmc_pwr_init(struct stm32_sdmmc_priv *priv)
 	return 0;
 }
 
-static int stm32_sdmmc_pwr_uninit(struct stm32_sdmmc_priv *priv)
+static int stm32_sdmmc_pwr_off(struct stm32_sdmmc_priv *priv)
```

Comment:
`stm32_sdmmc_pwr_off()` return status is never checked. If so, it would be better the function returns void.
Ditto for `stm32_sdmmc_card_detect_uninit()`.